ferolino_jasmin/country.html

```diff
@@ -0,0 +1,14 @@
+<body>
+    <h1>Countries of the World</h1>
+    <input type="text" id="countryInput" placeholder="Enter a country">

```

Comment:
### IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies to lines 6, and 9.